README.md

```diff
@@ -243,6 +244,27 @@ You may want to check other ways of authentication
 Doorkeeper support multiple languages. See language files in
 [the I18n repository](https://github.com/doorkeeper-gem/doorkeeper-i18n).
 
+### Rake Tasks
+
+If you are using `rake`, you can load rake tasks provided by this gem, by adding
+the following line to your `Rakefile`:
+
+```ruby
+Doorkeeper::Rake.load_tasks
```

Comment:
Alternatively we could load all the rake tasks when requiring `doorkeeper`, but I feel that this would force people to use rake, when it is not a strict dependency of this gem.

Reply:
Yep, everything is OK